src/components/paragraphs/sum-calculator/sum-calculator.tsx

```diff
@@ -87,12 +87,19 @@ const SumCalculatorParagraph = ({costPerStudentTypes, appCost, progCosts, i20Cos
     >
       <div className="mx-auto flex max-w-7xl flex-col gap-20 pb-72">
         <div onFocus={onSelectFocus}>
+          <label
```

Comment:
Just use a `<div>` instead of a `<label>`
![image](https://github.com/SU-SWS/summer-nextjs/assets/7185045/f7cb87a4-54f3-4897-b67b-2c5131b2ac23)